package.json

```diff
@@ -1,7 +1,5 @@
 {
   "name": "react-native-country-picker-modal",
-  "version": "2.0.0",
-  "description": "react-native country picker",
```

Comment:
I think the point was not to *remove* the lines, it was to remove them from the diff - to *not change* the lines
Then the diff should just purely be removing peerDependencies and updating react-native-web

From the perspective of a maintainer, any PR that touches the package.json metadata (not the dependency versions - but the name, version of package etc) is illegal, we can't merge them

Reply:
Yeah. Sorry. I made a mistake. I thought editing here removes the changes, not the lines.